cmd/bb_runner/main.go

```diff
@@ -111,6 +111,15 @@ func main() {
 		}
 		cleaners = append(cleaners, cleaner.NewDirectoryCleaner(directory, d))
 	}
+
+	if configuration.RunCommandCleaner != nil {
```

Comment:
Just as a safety belt, could you change this to:
```go
if len(configuration.RunCommandCleaner) > 0 {
```
I'm pretty sure that `proto.Unmarshal()` leaves empty lists as nil, but we can't be too sure about that. A non-nil empty list would cause us to index the list out of bounds.